core/src/main/kotlin/com/justai/jaicf/logging/ConversationLogObfuscator.kt

```diff
@@ -0,0 +1,32 @@
+package com.justai.jaicf.logging
+
+import com.justai.jaicf.api.BotRequest
+
+
+/**
+ * Base interface for logging content obfuscators.
+ * Implementations of this interface should be used to hide sensitive data during logging in [ConversationLogger].
+ *
+ * @see com.justai.jaicf.logging.ConversationLogger
+ * @see com.justai.jaicf.logging.ConsoleConversationLogger
+ * */
+interface ConversationLogObfuscator {
+
+    /**
+     * Obfuscates user input from [BotRequest] in [ConversationLogger] implementation.
+     *
+     * @param loggingContext current request's [LoggingContext]
+     *
+     * @return obfuscated input
+     * */
+    fun obfuscateInput(loggingContext: LoggingContext): String
+
+    /**
+     * Obfuscates bot reactions in [ConversationLogger] implementation.
+     *
+     * @param loggingContext current request's [LoggingContext]
+     *
+     * @return list of obfuscated [LoggingReaction]
+     * */
+    fun obfuscateReactions(loggingContext: LoggingContext): MutableList<LoggingReaction> = loggingContext.reactions
```

Comment:
Because `LoggingContext` contains mutable list of reactions, which can be appended on every reaction. 
If we make loggingContext.reactions immutable, we'd have to create new logging context on every reaction. Or store a mutable list of reactions somewhere else. 
```kotlin
loggingContext.copy(reactions = newReactions)
```

I found these solutions bad, therefore it returns mutable list.